gym_pybullet_drones/envs/BaseAviary.py

```diff
@@ -338,7 +338,6 @@ def step(self,
                                                           ) for i in range(self.NUM_DRONES)]
         #### Save, preprocess, and clip the action to the max. RPM #
         else:
-            self._saveLastAction(action)
```

Comment:
Are there not any dependencies on the self.last_action item in other aviaries?

Reply:
As far as I can tell, there are no reads nor writes of `self.last_action` in any of the aviaries in branch `dev-sb3` (the motivation for removing this is that method `self._saveLastAction(action)` was only compatible with actions of size 4, like RPMs and incompatible with other action spaces)